modules/oc-mirror-updating-registry-about.adoc

```diff
@@ -0,0 +1,31 @@
+// Module included in the following assemblies:
+//
+// * installing/disconnected_install/installing-mirroring-disconnected.adoc
+
+:_content-type: CONCEPT
+[id="oc-mirror-updating-registry-about_{context}"]
+= About updating your mirror registry content
+
+When you run the oc-mirror plug-in again, it generates an image set that only contains new and updated images since the previous execution. Because it only pulls in the differences since the previous image set was created, the generated image set is often smaller and faster to process than the initial image set.
+
+// TODO: Confirm that this statement is still accurate about only containing new and updated images, or did that change ^
```

Comment:
@jpower432 Just checking - is the above statement accurate still?

Reply:
@bergerhoffer This is correct.